web/src/AdapterListPage.js

```diff
@@ -32,7 +32,7 @@ class AdapterListPage extends BaseListPage {
       createdTime: moment().format(),
       type: "Database",
       host: "localhost",
-      port: 3306,
+      port: "3306",
```

Comment:
Isn't port int type?

Reply:
Has change to string type, it is more flexiable than int.